srcpkgs/glibc/template

```diff
@@ -98,14 +96,10 @@ do_configure() {
 	SHELL=/bin/bash ../configure ${configure_args} \
 		--bindir=/usr/bin --sbindir=/usr/bin \
 		--libdir=${_libdir} --libexecdir=${_libdir} \
-		--enable-add-ons=libidn \
 		--enable-multi-arch --enable-bind-now \
-		--disable-profile --enable-kernel=3.2.0 \
-		--enable-stack-guard-randomization \
-		--without-selinux --without-cvs --without-gd \
-		--disable-lock-elision \
```

Comment:
Is lock elision disabled by default now or why was this removed?

Reply:
https://github.com/bminor/glibc/commit/07ed18d26a342741cb25a4739158c65ed9dd4d09

elison tunables was added which depends on lock elison so the ability to disable lock elison was removed in 2017.